ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -498,3 +499,118 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def check_rebalance_occur_after_expand(old_osds):
+    """
+    The function check data balance is initiating after add capacity to cluster. The function need 2 out 3 rounds
+    the old osd that had data will be decreased and new osd data will be increased.
+    Args:
+        old_osds: list of old [osd.0, osd.1...]
+
+    """
+    from time import sleep
+    from ocs_ci.ocs.exceptions import ClusterUtilizationNotBalanced
+    from ocs_ci.ocs.cluster import get_osd_utilization
+    num_of_fails_old_osd_increase = 0
+    num_of_fails_new_osd_decrease = 0
+    for num in range(1, 4):
```

Comment:
Why 4? 
Please consider doing this is in a `while` loop with a defined loop exit, like timeout

Reply:
Changed